bin/ensureServiceUser

```diff
@@ -377,7 +377,11 @@ function wrapAction(actionFunc, serviceName, options) {
 
     const log = new werelogs.Logger(process.argv[1]).newRequestLogger();
     const iamClient = createIAMClient(options.parent);
-    const stsClient = createSTSClient(options.parent);
+    let stsClient;
+
+    if (options.stsEndpoint) {
+        stsClient = createSTSClient(options.parent);
+    }
```

Comment:
Is it enough not to create the client? Shall we return an error here if the policyFile is defined, to avoid crashing in the middle of the `buildServiceUserForInAccountAssumeRoleHandlers` function?

Reply:
I can add a check but this is not supposed to happen anyway